docs/get-started/Dockerfile.cuda

```diff
@@ -0,0 +1,89 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
are we wanting to put the nvidia copyright on this?
 * Copyright (c) 2020, NVIDIA CORPORATION.

Reply:
Updated license